IFC4x3/Properties/d/DistanceFromTensionerToGround_3xKupe5RT6XRnMcxYkd5Qt/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DistanceFromTensionerToGround_3xKupe5RT6XRnMcxYkd5Qt" Name="DistanceFromTensionerToGround" UniqueId="fb538ce8-15b7-4685-bc56-9bb8ae9c56b7" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Would this be better served if extracted from geometry?

Reply:
seconded

Reply:
@fhulin

Reply:
After discussing with domain, this is a design criteria giving reference distance value linked with temperature.

Reply:
08487af509e4319afa5d187d589ae5dfd92de558